app/javascript/src/Users/components/FeaturesFieldset.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import React from 'react'
+
+import { getFeatureName, canFeatureSetServicePermissions } from 'Users/utils'
+
+import type { Feature, AdminSections } from 'Users/types'
+
+type Props = {
+  features: Feature[],
+  selectedSections?: AdminSections[],
+  areServicesVisible?: boolean,
+  onAdminSectionSelected: AdminSections => void
+}
+
+/**
+ * Contains a set of Features that can be checked by the user.
+ * @param {Feature[]} features - An array containing all available Features in the fieldset.
```

Comment:
Nice that you are adding documentation, don't stop there and space them so they are aligned and easy to read ;)